plugins/lsp/conform-nvim.nix

```diff
@@ -69,15 +69,23 @@ in {
         '';
 
       formatAfterSave =
-        helpers.defaultNullOpts.mkNullable (types.submodule {
-          options = {
-            lspFallback = mkOption {
-              type = types.bool;
-              default = true;
-              description = "See :help conform.format for details.";
-            };
-          };
-        })
+        helpers.defaultNullOpts.mkNullable
+        (
+          with types;
+            either
+            str
```

Comment:
please use `with nixvimTypes; either [strLuaFn ....]` so that it is documented correctly.

Reply:
I've applied this change in saveOnSave as well